composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "adriansuter/twig-cache-busting",
+    "name": "duneyr/twig-cache-busting",
```

Comment:
The package name should be left as is.

Reply:
Sorry!!!!! Wrong branch!!!! I'll change it!!!